build/wix/build32.bat

```diff
@@ -0,0 +1,39 @@
+@echo off
```

Comment:
To avoid duplication, can you change these to a single wix/build.bat that takes the bit width as an argument? It should also take WINLIB_PATH as an argument, or assume it's already set in the environment, since scons does so.